src/index.ts

```diff
@@ -0,0 +1,19 @@
+import express, { Application, Response, Request } from "express";
+import dotenv from "dotenv"
+import morgan from "morgan"
+dotenv.config();
+const app: Application = express();
+const port = process.env.PORT || 8000;
+
+app.use(morgan('dev'));
+app.use(express.json());
+
+console.log(`Server is Fire at http://localhost:${port}`);
+
+app.listen(port, () => {
+    console.log(`Start To listen port ${port}`);
+})
+
+app.get("/", (_req: Request, _res: Response) => {
+    _res.send("Hello This is IFoundApi, checkout /api To see the document.");
+})
```

Comment:
res args is used, so delete  '_' .

Reply:
![0d0dd86e62190868e612238387c4263b](https://github.com/HEKEPOIU/ifound_backend/assets/101374417/bc2f2576-f6ef-4ca3-b814-b4b4a8ffc122)

Reply:
resolve